docs/guides/location-service/fragments/setting-up-your-app-android.md

```diff
@@ -0,0 +1,210 @@
+## Overview
+
+[Amazon Location Service](http://aws.amazon.com/location) is a geolocation service that lets you add location information to your applications. With Amazon Location Service, you can build applications that provide maps and points of interest, track resources, trigger actions based on location, and convert street addresses into geographic coordinates. This guide provides you with the following information:
+
+- How to setup and configure your Android app with the Amazon Location Service SDK
+- How to add authentication to your application
+- Example of how to access the Amazon Location Service APIs
+
+To learn more about the service, see the [Amazon Location Service Developer Guide](https://docs.aws.amazon.com/location/latest/developerguide/).
+
+## Prerequisites
+
+When using the Amazon Location Service within your Android app, your application must target API level 16 or later.
+
+## Installing dependencies
+
+The following procedure defines how to install the necessary Amazon Location Service SDK dependencies in your Android app.
+
+**To install dependencies:**
+
+1. In Android Studio, expand **Gradle Scripts** in the project viewer and open **build.gradle (Module: YourApp).**
+1. In the **dependencies** block, add the following lines:
+
+    ```groovy
+    implementation 'com.amazonaws:aws-android-sdk-mobile-client:2.20.+'
+    implementation 'com.amazonaws:aws-android-sdk-location:2.20.+'
+    ```
+
+1. Click **Sync Now** in the notification bar above the file editor to update your project’s configuration. You have now successfully added the Amazon Location Service SDK dependencies to your Android app.
+
+## Adding authentication to your App using the Amplify CLI
+
+The Amplify Framework uses [Amazon Cognito](https://aws.amazon.com/cognito/) as its primary authentication provider. Amazon Cognito is a robust user directory service that handles user registration, authentication, account recovery, and other operations. Use the following procedure to add authentication to your app using Amazon Cognito and Amplify CLI. 
+
+1. Open a terminal window in Android Studio by clicking Terminal.
+1. Install the [Amplify CLI](https://docs.amplify.aws/cli/start/install) by running the following command:
+
+    ```bash
+    npm install -g @aws-amplify/cli
```

Comment:
As this is the Android guide, shall we use the new-and-improved procedure, instead? To wit, `npm` had been a nearly non-existent aspect of my Android development life, before working on Amplify.

```suggestion
    curl -sL https://aws-amplify.github.io/amplify-cli/install | bash && $SHELL
```

Reply:
This seems valid to me. Can we make this work for Windows users, too? I think there's a tabbed snippet.

Reply:
Addressed!